Description: Removes Ch 14 fom example channels.conf.terr, because the entry is
 invalid (reported as Ubuntu Bug #45721)
Author: Thomas Schmidt <tschmidt@debian.org>

Index: vdr/channels.conf.terr
===================================================================
--- vdr.orig/channels.conf.terr	2011-12-10 22:22:04.000000000 +0100
+++ vdr/channels.conf.terr	2011-12-11 01:49:01.000000000 +0100
@@ -9,7 +9,6 @@
 bid-up.tv (TV):561833:I0C23D0M64B8T2G32Y0:T:27500:6273:6274:0:0:14272:0:0:0
 CBBC (TV):505833:I0C34D0M16B8T2G32Y0:T:27500:620:621,622:0:0:4671:0:0:0
 Cbeebies (TV):529833:I0C34D0M16B8T2G32Y0:T:27500:201:401,402:0:0:16960:0:0:0
-Ch 14 (TV):481833:I0C23D0M64B8T2G32Y0:T:27500:2840:2841:2843:0:0:8800:0:0
 Ch 32 (TV):537833:I0C34D0M16B8T2G32Y0:T:27500:501:502,504:0:0:25984:0:0:0
 Ch 44 (TV):537833:I0C34D0M16B8T2G32Y0:T:27500:501:502,504:0:0:26048:0:0:0
 Channel 4 (TV):481833:I0C23D0M64B8T2G32Y0:T:27500:2827:2828,2830:0:0:8384:0:0:0
